manual/virtualdub/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,18 @@
+﻿$packageName = 'virtualdub'
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url = "file://$toolsDir/VirtualDub-1.10.4.zip"
+$url64 = "file://$toolsDir/VirtualDub-1.10.4-AMD64.zip"
+
+Install-ChocolateyZipPackage $packageName $url $toolsdir $url64
+
+# Unique names for each bitness
+$binName = "Veedub64.exe"
+If ( Get-OSArchitectureWidth -compare '32' ) { $binName = "VirtualDub.exe" }
+
+# Place shortcuts in appropriate location
+$ProgsFolder = [environment]::getfolderpath('Programs')
+If ( Test-ProcessAdminRights ) { $ProgsFolder = [environment]::getfolderpath('CommonPrograms') }
```

Comment:
I'm sorry to say, but using `CommonPrograms` is possible.
Scripts are meant to be PSv2 compatible (equivalent to .NET 2.0), and `CommonPrograms` isn't available on platforms with PSv2 only installed.